src/nupic/os/Timer.cpp

```diff
@@ -66,38 +66,6 @@ static UInt64 getCurrentTime()
   return (UInt64)(v.QuadPart) - initialTicks_;
 }
 
-#elif defined(NTA_PLATFORM_darwin)
```

Comment:
> Do you have any background on using high precision timers on OS X like this code does? Do we need high precision like this? Did we intentionally stop using this code block?

Scott - I agree with you. We shouldn't remove code without understanding why it is there, even if there is a bug in it. These high res timers are useful for debugging and optimizing Cells4. They are turned on using a #define when someone is actively working on the code.  See lines 44-68 of Cells4.cpp.

I don't think we should remove any functionality in this PR. This PR is just about renaming right?